src/commands/NFD.ts

```diff
@@ -125,7 +128,13 @@ class NFD {
   @Slash('view', { description: 'View an existing NFD.' })
   @SlashGroup('nfd')
   async view(
-    @SlashOption('name', { type: ApplicationCommandOptionType.String, required: true })
+    @SlashOption('name', {
+      type: ApplicationCommandOptionType.String,
+      required: true,
+      autocomplete: function (this: NFD, interaction: AutocompleteInteraction) {
+        this.nfdAutocomplete(interaction.user.id).then((choices) => interaction.respond(choices))
```

Comment:
This isn't quite right. People should be able to \view NFDs they don't own. Do we need a flag in the function for "return all". Maybe if the user id is null then it returns all NFDs that exist?

Reply:
This was to limit the amount of autocomplete entries, I have since changed it to max at 25 but update as they type. 

Also the user doesn't have to pick an autocomplete, they can do anything and it would still work.